Sources/tuist/tuist.docc/Tutorials/Tuist-Cloud/Tuist-Cloud-Enterprise/Tuist Cloud - Enterprise Deployment.tutorial

```diff
@@ -29,7 +29,19 @@
         We recommend establishing a continuous deployment pipeline that operates daily, pulling and deploying fresh images. This ensures you consistently benefit from the latest improvements. If preferred, you can synchronize with Tuist Cloud's release schedule and deploy weekly, coinciding with each new official release.
 
         Subsequent sections provide examples of deployments using popular cloud providers.
+
+        ### Docker
         
+        You can deploy your app using docker directly.
+        First, make sure to have postgres setup. You can also run the following command to start a postgres database:
+        ```sh
+        docker run --name tuist_cloud_postgres_container -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d postgres
```

Comment:
Is the current Docker image running Postgres? The command seems to imply that with the port and database name flags. I'd not recommend that because the storage is not persisted across runs unless you mount an external volume that contains the database. What if instead we tell them to set it up themselves and run this one passing the URL to it?

```bash
docker run --name tuist_cloud -e DATABASE_URL=****
```